types/mjml-react/index.d.ts

```diff
@@ -125,43 +130,62 @@ export class MjmlStyle extends React.Component<{ children?: string | undefined,
 export class MjmlTitle extends React.Component<{ children?: string | undefined }> { }
 
 // mj-accordion
-export class MjmlAccordion extends React.Component<MjmlAccordionElementProps> { }
+export interface MjmlAccordionProps {
+    children?: React.ReactNode;
+    border?: string | undefined;
+    containerBackgroundColor?: React.CSSProperties['backgroundColor'] | undefined;
+    fontFamily?: string | undefined;
+    iconAlign?: string | undefined;
+    iconHeight?: string | undefined;
+    iconPosition?: 'left' | 'right' | undefined;
+    iconUnwrappedAlt?: string | undefined;
+    iconUnwrappedUrl?: string | undefined;
+    iconWidth?: string | undefined;
+    iconWrappedAlt?: string | undefined;
+    iconWrappedUrl?: string | undefined;
+}
+
+export class MjmlAccordion extends React.Component<MjmlAccordionProps & PaddingProps & ClassNameProps> { }
 
 export interface MjmlAccordionElementProps {
     children?: React.ReactNode;
+    border?: string | undefined;
+    backgroundColor?: React.CSSProperties['backgroundColor'] | undefined;
     fontFamily?: string | undefined;
     iconAlign?: string | undefined;
-    iconWrappedUrl?: string | undefined;
-    iconWrappedAlt?: string | undefined;
+    iconHeight?: string | undefined;
+    iconPosition?: 'left' | 'right' | undefined;
```

Comment:
A couple of props missing. Added here in line with MJML docs: https://documentation.mjml.io/#mj-accordion-element

Reply:
It looks to me like `border` is the only thing new here. All the others a re-ordered. Can you revert the re-orders?